symposion/speakers/forms.py

```diff
@@ -9,14 +9,14 @@
 
 class SpeakerForm(forms.ModelForm):
 
-    sessions_preference = forms.ChoiceField(
-        widget=forms.RadioSelect(),
-        choices=Speaker.SESSION_COUNT_CHOICES,
-        required=False,
-        help_text=_(u"If you've submitted multiple proposals, please let us "
-                    u"know if you only want to give one or if you'd like "
-                    u"to give two talks.")
-    )
+#    sessions_preference = forms.ChoiceField(
```

Comment:
Should this have a companion migration to remove the column, or is there some expectation that his may be restored at some point?